README.md

```diff
@@ -278,3 +278,42 @@ jsonPayload.verb="stacktraceanalyzer"
 3. `stack_trace_interval_seconds` signifies the duration in seconds between each stack trace collection event. Setting `stack_trace_interval_seconds: 600` will collect the stack traces every 600 seconds (10 minutes).
 
 Here is the related PyPI package: https://pypi.org/project/cloud-tpu-diagnostics.
+
+## Ahead of Time Compilation (AOT)
+To compile your training run ahead of time, we provide a tool `train_compile.py`. This tool allows you to compile the main `train_step` in `train.py` for target hardware (e.g. a large number of v5e devices) without using the target hardware, and instead you may use only a single VM from a different family, e.g. a v4-8. This compilation helps with two main goals:
```

Comment:
"from a different family or even not having an accelerator"

Reply:
Done.